docs/installingNative.md

```diff
@@ -20,7 +20,13 @@ Most of these prerequisites can be installed using the package manager of your O
     On Ubuntu, use `apt` for the main prerequisites:
 
     ```sh
-    sudo apt install git gcc libc6-dev graphviz graphviz-dev redis6 postgresql16 postgresql16-server postgresql16-contrib
+    sudo apt install git gcc libc6-dev graphviz libgraphviz-dev redis postgresql postgresql-contrib
```

Comment:
Assuming the changes here are tested; I don't have a Windows/Ubuntu machine to test with.

Reply:
Yep, tested on my WSL machine.